common-arcana.lic

```diff
@@ -406,6 +406,7 @@ module DRCA
       .spell_data
       .select { |_name, properties| properties['cyclic'] }
       .select { |name, _properties| DRSpells.active_spells.keys.include?(name) }
+      .reject { |name| get_settings.cyclic_no_release.include?(name) }
```

Comment:
Calling `get_settings` in a `common-*` file is undesirable and should be avoided if possible because it means we'll only ever pick up the setting as defined in the main YAML file - it can't be overriden in sub-YAML files. We should try to pass either the specific setting or settings itself into this function.

Reply:
Additionally, the `release_cyclics` method needs an argument to force releasing them regardless the user's setting. For example, when `combat-trainer` decides it's time to cast a new cyclic spell then it must release the current one.

Reply:
Hah, yes, I hadn't considered that scenario, and beyond that, the new functionality @Hiinky just added to release cyclics on low-mana would also need to be updated. Note, however, that since there can only ever be one cyclic cast at a time, even for a bard... it would be user error to configure a spell to not be released by scripts, and expect a different cyclic to be cast. Segue functionality for bards wouldn't be affected.

Reply:
@KatoakDR and I chatted about this. See the current version. Once this is in, we can go fix the various scripts that might want to use this. The main focus is really T2/Hunting-Buddy, which I'll look into in a couple of days after this is merged in.